client/views/actions/topic.tpl

```diff
@@ -1,4 +1,4 @@
-{{#if from}}
+{{#if from.id}}
```

Comment:
User object has no IDs.

Reply:
Ah, crap, that's why I shouldn't commit code that late at night. Do you know which property of `User` we could reliably check? Nick? Mode? Introduce an `id`? :man_shrugging:

Reply:
`nick` should be good enough.

Reply:
Done.